packages/functions/src/index.ts

```diff
@@ -40,7 +40,7 @@ export const getBookedDonationsInHospital = handler(
 );
 
 // Donor
-export const bookAppointment = handler(bookAppointmentHandler);
+export const donorBookAppointment = handler(donorBookAppointmentHandler);
```

Comment:
You can't change the API name. 
This will break all old versions calling this API.

Reply:
fixed

Reply:
Fixed